docs/modules/ROOT/pages/contributors/secret-rotation.adoc

```diff
@@ -23,3 +23,13 @@ and regenerate the token for the appropiate robot.
 `kubearchive-renovate`. Go to
 link:https://github.com/organizations/kubearchive/settings/apps/kubearchive-renovate[the kubearchive-renovate app page]
 and generate a new client secret.
+
+== KubeArchive Repository
+
+* `PUSH_TO_MAIN_DEPLOY_KEY`: SSH private key used to push to the default branch from
```

Comment:
Just a suggestion...

Based on the content, you might want to make this a description list instead of a bullet list. But, if you do, you'll likely have to update the list above this as well.

```suggestion
`PUSH_TO_MAIN_DEPLOY_KEY`:: SSH private key used to push to the default branch from
```

Reply:
I tested it with a description list and I like it better. Changed.